engine-tests/src/test/java/org/terasology/persistence/internal/StorageManagerTest.java

```diff
@@ -116,12 +100,12 @@ static void createFolder() throws IOException {
     @BeforeEach
     public void setup() throws Exception {
         super.setup();
-        JavaArchive homeArchive = ShrinkWrap.create(JavaArchive.class);
-        FileSystem vfs = ShrinkWrapFileSystems.newFileSystem(homeArchive);
-        PathManager.getInstance().useOverrideHomePath(temporaryFolder.toPath());
+        Path tempHome = Files.createTempDirectory("terasology-smt", (FileAttribute<?>[]) null);
+        tempHome.toFile().deleteOnExit();
+        PathManager.getInstance().useOverrideHomePath(tempHome);
```

Comment:
this extends TerasologyTestingEnvironment, which already does this in `beforeAll`. But I guess because these are storage tests, it wants to change to a clean directory `beforeEach` instead?